src/index/LocatedTriples.h

```diff
@@ -150,6 +150,10 @@ class LocatedTriplesPerBlock {
   std::vector<LocatedTriples::iterator> add(
       ql::span<const LocatedTriple> locatedTriples);
 
+  // Removes the given `LocatedTriple` from the `LocatedTriplesPerBlock`.
+  //
+  // NOTE: `updateAugmentedMetadata()` must be called to update the block
+  // metadata.
```

Comment:
Just a comment:
It is currently inconsistent that `add` has a batch interface, but the corresponding `erase` function only deletes a single triple. I know that the difference is, that we cannot easily get a good type for `all erase operations for a single permutation` due to the data layout, and that this design will probably be iterated in the future, but still.

Reply:
Yes. I would also have preferred a batch delete (and played with that), because this allows `LocatedTriples` to ensure that the metadata is updated. But also decided against it for now, because of the data layout.